docs/modules/operation/pages/deep-dive/service-assurance/configuration.adoc

```diff
@@ -1,40 +1,40 @@
 
 [[ga-pollerd-configuration]]
-= Pollerd Configuration
+= Configure Pollerd
 
-.Configuration and log files related to Pollerd.
+.Configuration and log files related to Pollerd
 [options="header, autowidth"]
 [cols="1,2"]
 |===
 | File
 | Description
 
 | $OPENNMS_HOME/etc/poller-configuration.xml
-| Configuration file for monitors and global daemon configuration
+| Configuration file for monitors and global daemon configuration.
 
 | $OPENNMS_HOME/logs/poller.log
-| Log file for all monitors and the global Pollerd
+| Log file for all monitors and the global Pollerd.
 
 | $OPENNMS_HOME/etc/response-graph.properties
-| RRD graph definitions for service response time measurements
+| RRD graph definitions for service response time measurements.
 
 | $OPENNMS_HOME/etc/events/opennms.events.xml
```

Comment:
I don't know that this is really relevant as a deep dive item.  It's extremely rare (and not recommended) to modify the internal OpenNMS events.  In the same vein, I don't know that the response-graph property file is worth mentioning here.
```suggestion
| $OPENNMS_HOME/etc/events/opennms.pollerd.events.xml
```